### ISSUE TITLE
Nav Color Contrast 

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#179]
Contrast between foreground text and background must be increased to meet WCAG AA standards.

<img width="1097" alt="Screen Shot 2020-04-14 at 11 51 15 AM" src="https://user-images.githubusercontent.com/31868510/79245661-61b0ab00-7e46-11ea-9990-b45d557bf5d9.png">

Element has insufficient color contrast of 3.96 (foreground color: #ffffff, background color: #7c7bad, font size: 16.5pt (22px), font weight: normal). Expected contrast ratio of 4.5:1 to meet WCAG AA
